10.0/s2i/bin/run

```diff
@@ -1,2 +1,12 @@
 #!/bin/bash
-exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0
+
+run_wildfly() {
+  if [ "$DEV_MODE" == true ]; then
+    exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0 -Dhot.deploy.enabled=true -DENABLE_JPDA=true
+  else
+    exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0
+  fi
+}
+
+run_wildfly
```

Comment:
the nodejs example calls the function from two places so it avoids in-lining the same block of logic twice.

if you're only invoking it once, the function doesn't save anything.

Though to be honest i'm not even sure what that sti-nodejs image is doing w/ the first invocation, that should probably be removed.  /usr/src/app isn't a directory that's going to exist ever.